prov/efa/src/rdm/efa_rdm_pke_rtm.c

```diff
@@ -1216,7 +1217,8 @@ ssize_t efa_rdm_pke_proc_matched_longread_rtm(struct efa_rdm_pke *pkt_entry)
 	efa_rdm_tracepoint(longread_read_posted, rxe->msg_id,
 		    (size_t) rxe->cq_entry.op_context, rxe->total_len);
 
-	err = efa_rdm_ope_post_remote_read_or_queue(rxe);
+	err = efa_rdm_pke_post_remote_read_or_nack(ep, pkt_entry, rxe);
+
 	if (err == -FI_ENOMR) {
```

Comment:
Should you merge this block to `efa_rdm_pke_post_remote_read_or_nack` ? Similar comments for other places.